experiment.py

```diff
@@ -899,7 +899,7 @@ def train(conf: TrainConfig, gpus, nodes=1, mode: str = 'train'):
 
     tb_logger = pl_loggers.WandbLogger(save_dir=conf.logdir,
                                        project='anomaly',
-                                       entity='jphwa',
+                                       entity='jy2k16',
```

Comment:
Would be great if you or @evaprakash can create a W&B "academic team" so we can see/edit runs all in one project